src/storybook/stand-alone-documentaion/welcome/other-contributors-list/other-contributors-list.jsx

```diff
@@ -47,23 +48,56 @@ const STATIC_CONTRIBUTERS = [
   {
     name: "LeanyLabs",
     href: "https://github.com/LeanyLabs"
+  },
+  {
+    name: "Hadas Farhi",
+    href: "https://github.com/hadasfa"
   }
 ];
 
+async function getContributors(page) {
+  let request = await fetch(
+    `https://api.github.com/repos/mondaycom/monday-ui-react-core/contributors?per_page=100&page=${page}&order=desc`,
+    {
+      method: "GET",
+      headers: {
+        "Content-Type": "application/json"
+      }
+    }
+  );
+
+  return await request.json();
+}
+
+async function getAllContributors() {
+  let contributors = [];
+  let page = 1;
+  let list;
+  do {
+    list = await getContributors(page++);
+    contributors = contributors.concat(list);
+  } while (list.length > 0);
+
+  return contributors;
+}
```

Comment:
how long does it take? how many pages there are?

this is something that can be done with `Promise.all` to run the fetches on parallel (collect promises and run them with Promise.all)

If it takes almost no-time, then it's cool

Reply:
104 contributors - 2 pages. Seems that it's working almost in no-time

Reply:
so cool for now 💪🏼